### PR TITLE
Feature/standalone pages

### PR DESCRIPTION
**Pull request by njseeto**
### Add standalone pages to new service generator
We need to have the ability to add standalone pages that are not part of
the flow. For now these are just the footer pages; cookies,
accessibility and privacy. They need to be there when a new service is
generated.

Instead of creating a standalone specific generator we adjusted the new
page generator to also be able to return just the page metadata instead
of returning the entire updated service metadata like it does when a
standard flow page is added.

### Make the standalone pages editable 
Configured the update and destroy methods in the MetadataUpdater to cater for standalone pages.
This PR allows the functionality to delete a standalone page, although this will not be surfaced in the UI so users will not be able to delete standalone pages.

### Rubocop corrections

Co-authored by: Brendan Butler <brendan.butler@digital.justice.gov.uk>

Screenshot 2021-04-12 at 16 37 39

### CODE REVIEW

File: app/generators/new_service_generator.rb [hunk 1]
```diff
@@ -17,5 +17,24 @@ def to_metadata
...
+  end
+
+  private
+
+  def footer_pages
+    I18n.t('footer').map do |attributes|
+      metadata = NewPageGenerator.new(
+        page_type: 'standalone',
+        page_url: attributes[:url]
+      ).to_metadata
```
Comment (tomas-stefano) [inline app/generators/new_service_generator.rb:32]: Do the standalone pages need an UUID?

File: app/generators/new_page_generator.rb [hunk 2]
```diff
@@ -1,17 +1,19 @@
 class NewPageGenerator
   include ActiveModel::Model
+  include ApplicationHelper
```
Comment (tomas-stefano) [inline app/generators/new_page_generator.rb:3]: Is the `strip_url` the reason of the included module?

**Review by tomas-stefano — DISMISSED** [review]
🚀 🚀 🚀  Awesome work! Left some minor questions!

File: app/generators/new_page_generator.rb (hunk 2 above)
Comment (brenetic) [reply app/generators/new_page_generator.rb:3]: yeah unfortunately so

File: app/generators/new_service_generator.rb (hunk 1 above)
Comment (tomas-stefano) [reply app/generators/new_service_generator.rb:32]: So with my changes from main, things would change on the UUID that can lead to a bug here. Now we need to pass the UUID to the new page generator.

Comment (brenetic) [reply app/generators/new_service_generator.rb:32]: ah yes. forgot about that. will update

**Review by tomas-stefano — APPROVED** [review]

**Pull request merged**